backend/src/main/kotlin/trie/Trie.kt

```diff
@@ -0,0 +1,53 @@
+package trie
```

Comment:
I überleg grad ob ma die klassen in dem package ńed a bissi auf die anderen verteilen sollen? 
weil `Trie` wär ja eigentlich a model, und die anderen 2 würden gut zu `utils/trie` oder so passen? weil bis jetzt basieren die packages ja ah ned auf die features - wobei ma natürlich ah gern alles so refactoren können wennst magst :) jetzt gehts nu leicht :D

Reply:
i würd des ned im model sehn, weil des andere im model is ja des Place was ans frontend gschickt wird. den trie will i echt ned ans frontend gebn :P aber joa, generell könn mas gern woanders hinschieben wennst willst

Reply:
habs jz einfach mit util package gwrapped :)